students/meet_shah/lesson08/Circle.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import math
+
+class Circle:
+
+    def __init__(self, radius=0):
+        self._radius = radius
+
+
+    @property
```

Comment:
no need to make a property for radius -- unless you really want it to be read-only.

just use a regular attribute.

try testing re-setting the radius -- it will fail -- was that on purpose?